snet_cli/commands.py

```diff
@@ -38,11 +37,25 @@ def _ensure(self, condition, message):
 
     def _printout(self, message):
         if self.out_f is not None:
-            print(message, file=self.out_f)
+            try:
+                message = str(message) + "\n"
+                self.out_f.write(message)
+            except UnicodeEncodeError as e:
+                if isinstance(self.out_f, type(sys.stdout)):
```

Comment:
Why do you need this if?

Reply:
so you should use hasattr function

Reply:
Ok.
Is there a reason to use `hasattr(self.out_f, "buffer")` or can I use `getattr(self.out_f, "buffer", None)`?

Reply:
```hasattr``` is a little bit cleanear because if attribute exists but it is ```None``` then ```getattr``` will return None, but hasttr will return true... But in this case of course result will be the same...

Reply:
I'm asking because `hasattr` will call `getattr` to return `True or False`.
But maybe using `hasattr` will make this part concise with other code parts.
I'm ok to use it BTW.
(I'm working with frontend now, so my head is far away from python. =/)